yesod-core/Yesod/Core/Class/Yesod.hs

```diff
@@ -366,6 +366,22 @@ sslOnlySessions = (fmap . fmap) secureSessionCookies
     setSecureBit cookie = cookie { setCookieSecure = True }
     secureSessionCookies = customizeSessionCookies setSecureBit
 
+-- | Helps defend against CSRF attacks by setting the SameSite attribute on
+-- session cookies to "Lax".
```

Comment:
I see what you are saying. As the spec is still evolving it's best if people do up to date research when considering whether or not to use this option. I'll think about how to clearly document that.